internal/notification/event.go

```diff
@@ -163,7 +163,7 @@ type gcsEvent struct {
 	ID              string            `json:"id"`
 	Name            string            `json:"name"`
 	Bucket          string            `json:"bucket"`
-	Generation      int64             `json:"generation,omitempty"`
+	Generation      string            `json:"generation,omitempty"`
```

Comment:
Go can do this for you:

```suggestion
	Generation      int64            `json:"generation,string,omitempty"`
```